test/VoteManager.js

```diff
@@ -35,13 +36,13 @@ contract('VoteManager', function (accounts) {
       // await stateManager.setEpoch(1)
       // await stateManager.setState(0)
       await functions.mineToNextEpoch()
-      await sch.transfer(accounts[3], 423000, { 'from': accounts[0]})
-      await sch.transfer(accounts[4], 19000, { 'from': accounts[0]})
-      await sch.approve(stakeManager.address, 420000, { 'from': accounts[3]})
-      await sch.approve(stakeManager.address, 19000, { 'from': accounts[4]})
+      await sch.transfer(accounts[3], new BN(423000).mul(new BN(10).pow(new BN('18'))), { 'from': accounts[0]})
```

Comment:
any specific reason for this change?

avoid changes unrelated to the assigned issue.

the pr should only solve the assigned issue. unrelated changes should be avoided.

Reply:
The previous code was causing the test cases in this file to fail. Making these changes made sure that the test cases were passing.